src/Core/Engine/Systems/AssetManager.hpp

```
@@ -33,7 +33,7 @@ AssetManager(const AssetManager&) = delete;
     ModelID Load(const std::string& directory);
 
     [[nodiscard]]
-    ModelID Load(std::unique_ptr<Model>& model);
+    ModelID Load(Model& model);
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)